geonode/layers/views.py

```diff
@@ -508,18 +509,67 @@ def layer_change_poc(request, ids, template='layers/layer_change_poc.html'):
 @login_required
 def layer_list(request):
     layer = Layer.objects.all()
+	filename = (str(datetime.datetime.now()) + "_layers.csv")
     response = HttpResponse(content_type='text/csv')
-    response['Content-Disposition'] = 'attachment; filename="geonode.csv"'
+    # response['Content-Disposition'] = 'attachment; filename="geonode_layers.csv"'  #ALTERNATIVE NAME IF DATETIME DOESNT WORK
```

Comment:
This was an extra line that should not have made it into the update, thank you for pointing it out.